github/ipfs-shipyard.yml

```diff
@@ -4666,29 +4550,16 @@ teams:
     description: People with write access to Javascript repositories
```

Comment:
:thought_balloon:  I don't think we've been using this group for day-to-day, superseded by [helia-dev](https://github.com/orgs/ipfs-shipyard/teams/helia-dev)

Reply:
The current version of a script assumes that a team is active as long as any user in that team remains active in at least one repository that team grants access to. So if we know of some teams that should not exist anymore, I think it'll be more efficient to flag and remove them proactively.